budget-squirrel/src/pages/Login/Login.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Login.scss'
+import BasicForm from '../../components/generic/BasicForm/BasicForm';
+import PasswordInput from '../../controls/PasswordInput/PasswordInput';
+
+export default function Login() {
+
+    const title = "Login";
+    const description = "Login in your account to view your budget details";
```

Comment:
In the wireframes, the login page doesn't have a description. I want us to mirror Nicks designs exactly on the actual site.